Makefile

```diff
@@ -10,7 +10,7 @@ getdeps:
 
 vet:
 	@echo "Running $@"
-	@go tool vet -atomic -bool -copylocks -nilfunc -printf -shadow -rangeloops -unreachable -unsafeptr -unusedresult *.go
+	@go vet -asmdecl -assign -atomic -bools -buildtag -cgocall -composites -copylocks -httpresponse -loopclosure -lostcancel -nilfunc -printf -shift -stdmethods -structtag -tests -unmarshal -unreachable -unsafeptr -unusedresult *.go
```

Comment:
Go is recommending moving to `go vet` only now which enables all of this and we can simplify this as `go vet ./...`